extensions/chocolatey-core.extension/extensions/Get-AvailableDriveLetter.ps1

```diff
@@ -0,0 +1,44 @@
+﻿<#
+.SYNOPSIS
+  Get a 'free' drive letter
+
+.DESCRIPTION
+  Get a not yet in-use drive letter that can be used for mounting
+
+.EXAMPLE
+  Get-AvailableDriveLetter
+
+.EXAMPLE
+  Get-AvailableDriveLetter 'X'
+  (do not return X, even if it'd be the next choice)
+
+.INPUTS
+  specific drive letter(s) that will be excluded as potential candidates
+
+.OUTPUTS
+  System.String (single drive-letter character)
+
+.LINK
+  http://stackoverflow.com/questions/12488030/getting-a-free-drive-letter/29373301#29373301
+#>
+function Get-AvailableDriveLetter {
+  param ([char[]]$ExcludedLetter)
```

Comment:
open for suggestions.. ?
* Exclude
* ExcludeThisLetter(s)
* ExcludeList

.. can't think of anything that doesn't sound totally awkward without 'Exclude' in the parameter name
* IgnoreLetter(s) 
* KeepDriveLetter(s)